src/components/EditCoursePage/index.jsx

```diff
@@ -57,29 +57,39 @@ class EditCoursePage extends React.Component {
     return `edit-course-form-${this.props.courseInfo.data.uuid}`;
   }
 
-  prepareCourseRuns(courseRuns, courseData) {
-    /* eslint-disable no-param-reassign */
-    courseRuns.forEach((courseRun) => {
-      if (courseRun.staff) {
-        courseRun.staff = courseRun.staff.map(staffer => staffer.uuid);
+  prepareSendCourseRunData(courseData, targetRun) {
+    const sendCourseRuns = [];
+    const modifiedCourseRuns =
```

Comment:
I would add back in the comment about how the backend doesn't like course runs that are in review.